examples/object-final-fields/object_final_fields.description

```diff
@@ -0,0 +1,6 @@
+// Fields of a class or an object constructor expression can be marked as `final`. Values can be set for such fields
+// either using initializer expressions (default values) or in the `init` function. A value cannot be set for such a
```

Comment:
Shouldn't this be init method?

Reply:
Yeah, my bad. Fixed here and in the bal file.